testsuite/integration/picketlink/src/test/java/org/wildfly/test/integration/security/picketlink/core/PartitionManagerProducerTestCase.java

```diff
@@ -71,8 +71,10 @@ public static WebArchive createDeployment() {
                    .addAsManifestResource(new StringAsset("Dependencies: org.picketlink.core meta-inf,org.picketlink.core.api meta-inf, org.picketlink.idm meta-inf\n"), "MANIFEST.MF")
                    .addClass(PartitionManagerProducerTestCase.class)
                     .addAsManifestResource(createPermissionsXmlAsset(
-                            new FilePermission("/tmp/pl-idm", "read,write,delete"),
-                            new FilePermission("/tmp/pl-idm/-", "read,write,delete")),
+                            new FilePermission("/tmp/pl-idm", "read,write,delete"),     // java.io.tmpdir cannot be used here as it has been changed for Arquillian
```

Comment:
I don't really get why java.io.tempdir cannot be used?
this kind of change is polluting the temp dir on CI if noting else.
We redirect all tmp dirs in CI to some "job workspace directories" so they can be properly cleaned up.